app/assets/javascripts/omni_nav/omni-nav-v2.js

```diff
@@ -193,15 +229,20 @@ var OmniNav2 = (function() {
       }
 
       // If no name selector, look for ID selector.
-      if (!$target.length) {
+      if ( !$target.length ) {
         $target = $(idSelector);
       }
 
+      // Still no target? Return or we get an error below.
+      if ( !$target.length ) {
+        return;
+      }
```

Comment:
Maybe this isn't such a good idea. It's swallowing an error that maybe we should see. If this is getting called and `$target` isn't present, it seems like it would be an issue. Perhaps another race issue?